### DISCUSSION TITLE
ImageGrabbed  event is not getting hit.

### DISCUSSION THREAD

**Discussion by IbrahimAliAsghar** [emgucv/emgucv#619] [General]
I have a created a basic application for getting image from webcam using the latest version of emgu cv, my webcam lights start, my capture object is also not null, but ImageGrabbed event is not getting hit. 
can you please tell me what I am doing wrong.
I am not getting any errors as well.

```
 public Form1()
        {
            InitializeComponent();

            _capture = new VideoCapture(0);


            _capture.ImageGrabbed += ProcessFrame;
            _frame = new Mat();
            if (_capture != null)
            {
                try
                {
                    _capture.Start();
                }
                catch (Exception ex)
                {
                    MessageBox.Show(ex.Message);
                }
            }
        }

```
```

 private void ProcessFrame(object sender, EventArgs arg)
        {
            if (_capture != null && _capture.Ptr != IntPtr.Zero)
            {
                _capture.Retrieve(_frame, 0);
                imageBox1.Image = _frame;
            }
        }
```

Environment : windows 10 x64, Visual Studio  2019

**Comment by emgucv** [comment]
You can call VideoCapture.IsOpen:
https://www.emgu.com/wiki/files/4.5.3/document/html/db1fad8e-1065-5269-f1d1-aaf2696cb89f.htm
and make sure the property is true after creating the VideoCapture object.

Besides of that, does our demo projects (in the .zip and .exe release files) that use camera work on your system?
